docs/azure/integrations-overview.md

```diff
@@ -1,39 +1,46 @@
 ---
 title: Azure integrations overview
 description: Overview of the Azure integrations available in the .NET Aspire.
-ms.date: 12/13/2024
+ms.date: 12/17/2024
 uid: dotnet/aspire/integrations/azure-overview
 ---
 
-<!--
+# .NET Aspire Azure integrations overview
+
+[Azure](/azure) is the most popular cloud platform for building and deploying [.NET applications](/dotnet/azure). The [Azure SDK for .NET](/dotnet/azure/sdk/azure-sdk-for-dotnet) allows for easy management and use of Azure services. .NET Aspire provides a set of integrations with Azure services, where you're free to add new resources or connect to existing ones. This article details some common aspects of all Azure integrations in .NET Aspire and aims to help you understand how to use them.
+
+## Add connection to existing Azure resources
 
-Somebody reading the overview is supposed to understand:
+.NET Aspire provides the ability to connect to existing Azure resources. Expressing connection strings is useful when you have existing Azure resources that you want to use in your .NET Aspire app. The <xref:Aspire.Hosting.ParameterResourceBuilderExtensions.AddConnectionString*> API is used with the app host's [execution context](../fundamentals/app-host-overview.md#execution-context) to conditionally add a connection string to the app model.
```

Comment:
This line sounds like the reader is supposed to already understand what AddConnectionString does. I think we should explain a couple of assumptions here:
1. Connecting to an existing azure resource is the same as connecting to an existing resource. We treat then like an external resource that the user knows how to configure.
2. We assume that role assignments have already been set up for the user. When you call AddConnectionString, the app host does not even know that the resource is an azure resource.
3. We should explain where to put these connection strings and how users can discover the format supported for each of the azure client integrations.
4. Don't use key vault for the sample, use storage. Keyvault is very confusing to use with AddConnectionString in this context since it's also a place to store secrets.

Reply:
Addressed in 06300832c8d2102a82d9b943d65e89b01f9e0318